### PR TITLE
[v2] windows: Add arm64 support

### LINKED ISSUE #979: [v2] Windows ARM64 support
**Is your feature request related to a problem? Please describe.**

I would like to bring wails to windows/arm64 and started working on the port. I got the sample project running on windows arm64 with two small patches to wails and to go-webview2.

I'll provide the PRs and hopefully we could get them merged 😄

### PR DESCRIPTION
**Pull request by stffabi**
Fix #979

### CODE REVIEW

File: v2/go.mod [hunk 1]
```diff
@@ -82,3 +82,5 @@ require (
 	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 )
+
+replace github.com/leaanthony/go-webview2 v0.0.0-20211022194343-1e4c8d4226f3 => ../go-webview2
```
Comment (stffabi) [inline v2/go.mod:86]: Will be removed as soon as the leaanthony/go-webview2#2 has been merged.

Comment (leaanthony) [reply v2/go.mod:86]: @stffabi Added a couple of qs to the other PR. Thanks 👍

Comment (stffabi) [reply v2/go.mod:86]: @leaanthony What do you mean by 'qs'? I couldn't find any comments on the other PR.

Comment (leaanthony) [reply v2/go.mod:86]: My bad @stffabi - the review was sat in draft 🤦

**Comment by stffabi** [comment]
@leaanthony I've updated the PR and bumped go-webview2.

**Comment by leaanthony** [comment]
I bow to your awesome work 🙏  LGTM!

**Comment by leaanthony** [comment]
I really want to update github actions to test v2 on different mac releases and now arm64 if possible. Some day :-)

**Comment by stffabi** [comment]
It's really a lot of fun to work on wails. I have to thank you for your awesome and hard work on wails. It's such a great project.

**Pull request merged**